traefik/config/traefik.toml

```diff
@@ -0,0 +1,6 @@
+logLevel = "{{cfg.log.level}}"
+
+{{~#if cfg.web.enable}}
```

Comment:
this isn't a blocker but just a note for the future. You can use `{{toToml cfg}}` and have `default.toml` share a similar structure to `traefik.toml.